virt/virtual_machines/advanced_vm_management/virt-vm-control-plane-tuning.adoc

```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="virt-vm-control-plane-tuning"]
+= Virtual machine control plane tuning
+
+include::_attributes/common-attributes.adoc[]
+:context: virt-control-plane-tuning
+
+OpenShift Virtualization offers the following tuning options at the control-plane level:
+
+* The `highBurst` profile, which uses fixed `QPS` and `burst` rates, to create hundreds of virtual machines (VMs) in one batch
+* Migration setting adjustment based on workload type
+
+
+toc::[]
+
+include::modules/virt-configuring-rate-limiters.adoc[leveloffset=+1]
```

Comment:
ToC is not rendering, possibly because it needs to be moved up. Idk how it works but the inclusion of item 6 in [this explanation](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) makes me think it's picky.

```suggestion
:context: virt-control-plane-tuning

toc::[]

OpenShift Virtualization offers the following tuning options at the control-plane level:

* The `highBurst` profile, which uses fixed `QPS` and `burst` rates, to create hundreds of virtual machines (VMs) in one batch
* Migration setting adjustment based on workload type

include::modules/virt-configuring-rate-limiters.adoc[leveloffset=+1]
```